indexer/editable_map_object.hpp

```diff
@@ -38,25 +38,37 @@ struct LocalizedName
   LocalizedName(int8_t code, string const & name);
   LocalizedName(string const & langCode, string const & name);
 
-  // m_code, m_lang and m_langName are defined in StringUtf8Multilang.
+  /// m_code, m_lang and m_langName are defined in StringUtf8Multilang.
   int8_t const m_code;
-  // Non-owning pointers to internal static char const * array.
+  /// Non-owning pointers to internal static char const * array.
   char const * const m_lang;
   char const * const m_langName;
   string const m_name;
 };
 
-// Class which contains vector of localized names with following priority:
-//  1. Names for Mwm languages
-//  2. User`s language name
-//  3. Other names
-// and mandatoryNamesCount - count of names which should be always shown.
+/// Class which contains vector of localized names with following priority:
+///  1. Names for Mwm languages
+///  2. User`s language name
+///  3. Other names
+/// and mandatoryNamesCount - count of names which should be always shown.
 struct NamesDataSource
 {
   vector<LocalizedName> names;
   size_t mandatoryNamesCount = 0;
 };
-  
+
+struct FakeName
+{
+  int8_t m_code;
```

Comment:
So, please, add a default ctor with default zero value for m_code. Better safe than sorry, you know.